### ISSUE TITLE
Error when getting a version later than 1.8.0 from gradle repo

### ISSUE THREAD

**Issue by joeha480** [i-net-software/SetupBuilder#46]
I suspect this is a problem with the gradle repo, but in case it is not:

When I try to update to a version beyond 1.8.0 I get a message similar to this
 > Could not resolve all dependencies for configuration ':classpath'.
   > Could not find gradle.plugin.de.inetsoftware:SetupBuilder:3.4.2.
     Searched in the following locations:
         https://plugins.gradle.org/m2/gradle/plugin/de/inetsoftware/SetupBuilder/3.4.2/SetupBuilder-3.4.2.pom
         https://plugins.gradle.org/m2/gradle/plugin/de/inetsoftware/SetupBuilder/3.4.2/SetupBuilder-3.4.2.jar

Clicking one of the above links results in an error such as this:
```xml
  <Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <Key>gradle.plugin.de.inetsoftware/SetupBuilder/3.4.2/2592beae79e48fff81594da9ee342e419916bf7e5b31e5e3ba1e3152c8fa0ddf/SetupBuilder-3.4.2.jar
  </Key>
  <RequestId>3D8EECBBC42FE438</RequestId>
  <HostId>
bjz9ux9bjYuplocRYXmGxHR9/QoZv5v0WYDPTUSvkAruMVWWtindchbzrVATVMWmK1gbD7GAeEs=
  </HostId>
  </Error>
```

**Comment by Horcrux7** [comment]
I think this is because you have set a not existing dependency. A component of gradle.plugin.de.inetsoftware does not exists. I recommended the new syntax if possible.

**Comment by joeha480** [comment]
Actually I get this issue with only this in build.gradle:
```gradle
plugins {
  id "de.inetsoftware.setupbuilder" version "3.4.2"
}
```
I could create a repo with this if you'd like...

**Comment by gamma** [comment]
There was a problem with the publish plugin that we used to send artifacts to plugin.gradle.org
It is fixed now and a new build 3.4.3 has been pushed.
